var data =
[{"data_format": 1, "description": "This Snakemake pipeline implements the GATK best-practices workflow", "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n", "full_name": "snakemake-workflows/dna-seq-gatk-variant-calling", "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmpun_hm24f/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmpun_hm24f/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmpun_hm24f/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpun_hm24f/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpun_hm24f/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpun_hm24f/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n", "stargazers_count": 121, "subscribers_count": 2, "topics": ["reproducibility", "snakemake", "sciworkflows"], "updated_at": 1610525750.0}, {"data_format": 1, "description": "RNA-seq workflow using STAR and DESeq2", "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n", "full_name": "snakemake-workflows/rna-seq-star-deseq2", "linting": "Lints for snakefile /tmp/tmpm2ra6915/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpm2ra6915/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpm2ra6915/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpm2ra6915/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 8, /tmp/tmpm2ra6915/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 32, /tmp/tmpm2ra6915/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 71, /tmp/tmpm2ra6915/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 173, /tmp/tmpm2ra6915/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 241, /tmp/tmpm2ra6915/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 276, /tmp/tmpm2ra6915/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n", "stargazers_count": 150, "subscribers_count": 5, "topics": ["snakemake", "snakemake-workflows", "ngs", "rna-seq", "reproducible-science"], "updated_at": 1609268872.0}]
